OrangeHRMTests/PageObjects/Pages/AdminPage.cs

```diff
@@ -0,0 +1,76 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Extensions.ExtensionMethods;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.PageObjects.Elements;
+
+namespace OrangeHRMTests.PageObjects.Pages
+{
+    public class AdminPage : BasePage
+    {
+        private MyWebElement AdminPageHeaderElementPartOne = new MyWebElement(By.XPath("//h6[@class='oxd-text oxd-text--h6 oxd-topbar-header-breadcrumb-module']"));
```

Comment:
some comments on AdminPageHeaderElementPartOne and AdminPageHeaderElementPartTwo
1. do you really need 2 parts? maybe you could get full title and assert it?
2. not the best locators you could write, I would rather try to use this one
![image](https://github.com/agrren/ReneiskiDiploma/assets/55654931/f9ce8fb7-320c-47b8-bf22-0e68ea063565)

Reply:
Fixed